src/main/java/com/simibubi/create/foundation/block/IHasBigOutline.java

```diff
@@ -0,0 +1,15 @@
+package com.simibubi.create.foundation.block;
+
+import net.minecraft.core.BlockPos;
+import net.minecraft.world.level.BlockGetter;
+import net.minecraft.world.level.block.state.BlockState;
+import net.minecraft.world.level.block.state.BlockBehaviour;
+import com.simibubi.create.content.trains.track.TrackBlock;
+import com.simibubi.create.content.decoration.slidingDoor.SlidingDoorBlock;
+
+/**
+ * Implementing this interface will allow you to have bigger outlines when overriding {@link BlockBehaviour#getInteractionShape(BlockState, BlockGetter, BlockPos)}
+ * <p>
+ * For examples look at {@link TrackBlock} and {@link SlidingDoorBlock}
+ */
+public interface IHasBigOutline { }
```

Comment:
Newly introduced Interfaces should no longer make use of the I-prefix.  
In this case though i think it's fine to keep. Either way, please rename the `Has` to `Have`